docs/source/getting-started/settlement.rst

```diff
@@ -0,0 +1,17 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Getting started 2 : Settlement
+##############################
+
+Non-intermediated example going from simple transfer to atomic DvP
+
+- Single Transfer (Alice -> Bob)
+- Atomic Dvp (Alice -> Bob and Bob -> Alice)
+
+  #. Instructing
+  #. Allocation
+  #. Approval
+  #. Settlement
+
+
```

Comment:
This is just a placeholder file that I created to get the Previous/Next-buttons to work in the right order. Are we fine with merging it like this for the time being, or does it need some content first?

Reply:
as this is still a wip, i would be fine with merging it. I actually already have some code to put in here, but need to write the text.